lectures/heavy_tails.md

```diff
@@ -1096,6 +1096,8 @@ $(0, \infty)$.
 
 The Pareto distribution is also heavy-tailed.
 
+Less formally, a **heavy-tailed** distribution is one that is not exponentially bounded (i.e. the tails are heavier than the exponential distribution). 
```

Comment:
@jstac not sure we should keep this but this is one "explanation" based on the wiki page. Thoughts?

Reply:
I think it's useful but maybe remove bold face because this is not the formal definition.

Reply:
thanks @jstac didn't scroll all the way to see this. 👍 

I will adjust now.